documentation/api/v2g.rst

```diff
@@ -0,0 +1,139 @@
+.. _v2g:
+
+Vehicle-to-grid
+---------------
+
+As a demonstration of how to construct a suitable flex model for a given use case, we consider a client using FlexMeasures to compute :abbr:`V2G (vehicle-to-grid)` schedules. In this example, the client is interested in the following:
```

Comment:
Here we should also link back to the place where the flex-model is actually introduced in full.